extensions/wikia/AdEngine/js/spec/lookup/prebid/bidHelper.spec.js

```diff
@@ -0,0 +1,57 @@
+/*global beforeEach, describe, it, modules, expect, spyOn*/
+describe('ext.wikia.adEngine.lookup.prebid.bidHelper', function () {
+	'use strict';
+
+	function getModule() {
+		return modules['ext.wikia.adEngine.lookup.prebid.bidHelper'](
+			mocks.bidHelper
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "mocks" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)